webrtc/RTCDataChannel-bufferedAmount.html

```diff
@@ -62,20 +62,19 @@
   0xe4, 0xb8, 0x96, 0xe7, 0x95, 0x8c,
   0xe4, 0xbd, 0xa0, 0xe5, 0xa5, 0xbd);
 
+for (const options of [{}, {negotiated: true, id: 0}]) {
+
+const datachannel = `${options.negotiated? "negotiated " : ""}datachannel`;
```

Comment:
Not the best of names. It's not a datachannel, it's a datachannel mode. What about calling it "mode"?